README.md

```diff
@@ -421,6 +421,7 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 * [Zsh](https://www.zsh.org) - Zsh is a shell designed for interactive use, although it is also a powerful scripting language. [![Open-Source Software][OSS Icon]](https://sourceforge.net/projects/zsh/) ![Freeware][Freeware Icon] [![Awesome List][awesome-list Icon]](https://github.com/unixorn/awesome-zsh-plugins#readme)
 * [xcodes](https://github.com/RobotsAndPencils/xcodes) - A command-line tool to install and switch between multiple versions of Xcode. [![Open-Source Software][OSS Icon] ![Freeware][Freeware Icon]](https://github.com/RobotsAndPencils/xcodes)
 * [xxh](https://github.com/xxh/xxh) - Bring your favorite shell wherever you go through the SSH. [![Open-Source Software][OSS Icon]](https://sourceforge.net/projects/zsh/) ![Freeware][Freeware Icon]
+* [fzf](https://github.com/junegunn/fzf) - 🌸 A command-line fuzzy finder. [![Open-Source Software][OSS Icon]](https://github.com/junegunn/fzf) ![Freeware][Freeware Icon]
```

Comment:
@Ian729 This is a command line tool application that can be PR below:

https://github.com/learn-anything/command-line-tools
https://github.com/herrbischoff/awesome-macos-command-line
https://github.com/herrbischoff/awesome-command-line-apps